src/asn.mli

```diff
@@ -332,6 +333,10 @@ module S : sig
   val int : int t
   (** [int] is ASN.1 [INTEGER], projected into an OCaml [int]. *)
 
+  val unsigned_integer : string t
+  (** [unsigned_integer] is ASN.1 [INTEGER], where the necessary two's
+      complement transformations are already applied. *)
+
```

Comment:
I added some more words. I think it is helpful to describe what happens if a negative ASN.1 INTEGER is encountered.
```suggestion
  val unsigned_integer : string t
  (** [unsigned_integer] is ASN.1 [INTEGER], where the necessary two's
      complement transformations are already applied. That is, it represents
      unsigned integers encoded as ASN.1 (signed) [INTEGER]s. Negative ASN.1
      [INTEGER]s are rejected with a parse error. *)

```